compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,32 @@
+# onecc-docker
+
+*onecc-docker* can expand `one-cmds` in more platforms than just Ubuntu 18.04/20.04.
+
+
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So, It is difficult for people in different environments to use our tools without using ubuntu18:04.
+
+To overcome this limitation, we provide _onecc-docker_ that runs using a docker so that users can use `one-cmds` more widely.
+
+Currently, `onecc-docker` is only supported in Linux environments.
```

Comment:
(optional) there was `different environments` in above and we can expect to use by Docker with other platforms than Linux. This line gives different story. Can we not use in Mac or Windows? or is it just because it is not free anymore?
My suggestion is to remove this line if not true.